<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeepLearningSegmentationGUI</class>
 <widget class="QWidget" name="DeepLearningSegmentationGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>222</width>
    <height>96</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QmitkTemplate</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="labelWarning">
     <property name="styleSheet">
      <string notr="true">QLabel { color: rgb(255, 0, 0) }</string>
     </property>
     <property name="text">
      <string>Please load a network!</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="buttonLoadTrainedNetwork">
     <property name="text">
      <string>Load network</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="buttonPerformImageProcessing">
     <property name="toolTip">
      <string>Do image processing</string>
     </property>
     <property name="text">
      <string>Run Segmentation</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
